src/Tizen.NUI/src/public/BaseComponents/Style/TextEditorStyle.cs

```diff
@@ -110,6 +116,18 @@ public class TextEditorStyle : ViewStyle
         internal static readonly BindableProperty GrabHandlePressedImageProperty = BindableProperty.Create(nameof(GrabHandlePressedImage), typeof(string), typeof(TextEditorStyle), String.Empty, propertyChanged: (bindable, oldValue, newValue) => ((TextEditorStyle)bindable).grabHandlePressedImage = (string)newValue,
             defaultValueCreator: (bindable) => ((TextEditorStyle)bindable).grabHandlePressedImage);
 
+        /// <summary> The bindable property of SelectionHandleImageLeft. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static readonly BindableProperty SelectionHandleImageLeftProperty = BindableProperty.Create(nameof(SelectionHandleImageLeft), typeof(PropertyMap), typeof(TextEditorStyle), null,
+            propertyChanged: (bindable, oldValue, newValue) => ((TextEditorStyle)bindable).selectionHandleImageLeft = (PropertyMap)newValue,
+            defaultValueCreator: (bindable) => ((TextEditorStyle)bindable).selectionHandleImageLeft);
+
+        /// <summary> The bindable property of SelectionHandleImageRight. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static readonly BindableProperty SelectionHandleImageRightProperty = BindableProperty.Create(nameof(SelectionHandleImageRight), typeof(PropertyMap), typeof(TextEditorStyle), null,
```

Comment:
Is this public like TextField and TextField and TextLabel?

Reply:
Same as above.